src/dtscalibration/datastore_utils.py

```diff
@@ -232,7 +244,25 @@ def merge_double_ended_times(ds_fw, ds_bw, verbose=True):
             if verbose:
                 print(f"Missing forward measurement beween {ds_bw.time.values[ind]} and {ds_bw.time.values[ind_next]}")
 
-    return ds_fw.isel(time=iuse_chfw), ds_bw.isel(time=iuse_chbw)
+    # throw out is dt differs from its neighbors
+    if verify_timedeltas:
+        dt = (ds_bw.isel(time=iuse_chbw).time.values - ds_fw.isel(time=iuse_chfw).time.values) /\
```

Comment:
Instead of the \ you can also do:
```py
dt = (
    (a + b)/
    c
)
```
which is generally the preferred way of breaking statements over multiple lines